instrumentation/jetty-httpclient/jetty-httpclient-9.2/library/src/main/java/io/opentelemetry/javaagent/instrumentation/jetty/httpclient/v9_2/JettyClientHttpAttributesExtractor.java

```diff
@@ -0,0 +1,150 @@
+package io.opentelemetry.javaagent.instrumentation.jetty.httpclient.v9_2;
+
+import io.opentelemetry.instrumentation.api.instrumenter.http.HttpAttributesExtractor;
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.eclipse.jetty.client.api.Request;
+import org.eclipse.jetty.client.api.Response;
+import org.eclipse.jetty.http.HttpField;
+import org.eclipse.jetty.http.HttpHeader;
+import org.eclipse.jetty.http.HttpVersion;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class JettyClientHttpAttributesExtractor extends HttpAttributesExtractor<Request, Response> {
+  private static final Logger LOG =
+      LoggerFactory.getLogger(JettyClientHttpAttributesExtractor.class);
+
+  private static final String[] httpFlavors = {"0.9", "1.0", "1.1", "2.0"};
+
+  @Override
+  @Nullable
+  protected String method(Request request) {
+    return request.getMethod();
+  }
+
+  @Override
+  @Nullable
+  protected String url(Request request) {
+    return request.getURI().toString();
+  }
+
+  @Override
+  @Nullable
+  protected String target(Request request) {
+    if (request != null) {
```

Comment:
Is this nullcheck needed? `request` should never be null here; also you don't have nullchecks in `url()` or `method()`